systemtest/apmservertest/server.go

```diff
@@ -449,6 +450,9 @@ func (s *Server) Close() error {
 	if err := interruptProcess(s.cmd.Process); err != nil {
 		s.cmd.Process.Kill()
 	}
+	// drain stderr even if there was no err so that the
+	// consumeStderr goroutine exits
+	io.ReadAll(s.Stderr)
```

Comment:
Should we close `s.Stderr`, rather than reading it? If the read end of the pipe is closed, then writes will fail. That should cause `consumeStderr` to exit more quickly, IIANM.

Reply:
That works too!